### ISSUE TITLE
Week #2: Improving the classical GANs

### ISSUE THREAD

**Issue by ericardomuten** [eraraya-ricardo/CERN-QGAN#3]
- [x] Re-train the DijetGAN generator with several GANs losses (cross-entropy loss, Wasserstein loss, etc.).
- [x] Re-create the DijetGAN model in PyTorch (previously implemented in TensorFlow).
- [x] Try several GANs metrics (chi-square, Wasserstein distance).

**Comment by VoicuTomut** [comment]
We should also rescale the data in the range[0,1].

**Comment by ericardomuten** [comment]
> 
> 
> We should also rescale the data in the range[0,1].

I think for the classical model, we can leave it as it is because it doesn't really matter which range we use.
But I will add it for the quantum case. Thanks!

**Comment by togtim-aws** [comment]
+ I'll try to adjust the PyTorch model according to the DijetGAN approach
